src/transport/Tuple.h

```diff
@@ -129,6 +131,27 @@ class Tuple : public Base
         return false;
     }
 
+    /**
+     * Recursive disconnect implementation iterating through transport members.
+     *
+     * @tparam N the index of the underlying transport to send disconnect to
```

Comment:
"@param", right?

Reply:
tparam is correct for template parameters.